tests/postgres_tests/test_indexes.py

```diff
@@ -217,6 +257,28 @@ def test_gin_parameters(self):
             editor.remove_index(IntegerArrayModel, index)
         self.assertNotIn(index_name, self.get_constraints(IntegerArrayModel._meta.db_table))
 
+    @skipUnlessDBFeature('has_bloom_index_support')
+    def test_bloom_index(self):
+        index_name = 'char_field_model_field_bloom'
+        index = BloomIndex(fields=['field'], name=index_name, length=80, columns=[4])
+        with connection.schema_editor() as editor:
+            editor.add_index(CharFieldModel, index)
+        constraints = self.get_constraints(CharFieldModel._meta.db_table)
+        self.assertEqual(constraints[index_name]['type'], BloomIndex.suffix)
+        self.assertEqual(constraints[index_name]['options'], ['length=80', 'col1=4'])
+        with connection.schema_editor() as editor:
+            editor.remove_index(CharFieldModel, index)
+        self.assertNotIn(index_name, self.get_constraints(CharFieldModel._meta.db_table))
```

Comment:
Can we add test for an index creation without optional parameters?

Reply:
Sure.